lib/stacks/eks-blueprint-stack.ts

```diff
@@ -249,4 +252,13 @@ export class EksBlueprint extends cdk.Stack {
 
         return vpc;
     }
+
+    /**
+     * Returns AddOn Id if defined else returns the class name
+     * @param addOn 
+     * @returns 
+     */
+    private getAddOnNameorId(addOn: spi.ClusterAddOn): string {
```

Comment:
Very minor:

1. `getAddOnNameOrId` - Or (capital)
2. It has not dependency on this, so does not have to be a member function. We can make it a regular function outside of Blueprint class and it will reduce verbosity.

Reply:
addressed, please review